graph-builder/src/plugins/github_openshift_secondary_metadata_scraper/plugin.rs

```diff
@@ -73,10 +94,20 @@ impl GithubOpenshiftSecondaryMetadataScraperPlugin {
         let url = github_v3::branches_url(&self.settings.github_org, &self.settings.github_repo);
 
         trace!("Getting branches from {}", &url);
-        let bytes = reqwest::Client::new()
-            .get(&url)
-            .header(reqwest::header::USER_AGENT, USER_AGENT)
-            .header(reqwest::header::ACCEPT, "application/vnd.github.v3+json")
+
+        let request = {
+            let request = reqwest::Client::new()
```

Comment:
Create Client once in `try_new` and reuse it here to reuse the connection to Github